doc/troubleshooting/howtoset_netsize.md

```diff
@@ -0,0 +1,139 @@
+# How to tune Fitting/embedding-net size ?
+
+Here are some test forms on fitting-net size tuning or embedding-net size tuning performed on several different systems.
+
+
+## Al2O3
+
+### Fitting net size tuning form on Al2O3: (embedding-net size: [25,50,100])
+
+Fitting-net size | Energy L2err(eV) | Energy L2err/Natoms(eV) | Force L2err(eV/Angstrom)
+---|---|---|---
+[240,240,240] |   1.742252e-02 |  7.259383e-05 |  4.014115e-02 
+[80,80,80]    |   1.799349e-02 |  7.497287e-05 |  4.042977e-02 
+[40,40,40]    |   1.799036e-02 |  7.495984e-05 |  4.068806e-02 
+[20,20,20]    |   1.834032e-02 |  7.641801e-05 |  4.094784e-02 
+[10,10,10]    |   1.913058e-02 |  7.971073e-05 |  4.154775e-02 
+[5,5,5]       |   1.932914e-02 |  8.053808e-05 |  4.188052e-02 
+[4,4,4]       |   1.944832e-02 |  8.103467e-05 |  4.217826e-02 
+[3,3,3]       |   2.068631e-02 |  8.619296e-05 |  4.300497e-02 
+[2,2,2]       |   2.267962e-02 |  9.449840e-05 |  4.413609e-02 
+[1,1,1]       |   2.813596e-02 |  1.172332e-04 |  4.781115e-02 
+[]            |   3.135002e-02 |  1.306251e-04 |  5.373120e-02 
```

Comment:
Extra question: does it mean it's possible to have no fitting network?

Reply:
Seems like it.

Reply:
`[]` means not hidden layer, but we have a linear output layer. This situation is equal to the linear regression.  

Please add a short note the explain the meaning of `"[]"`

Reply:
Done.